docs/guide/tutorial-8-using-ros.md

```diff
@@ -77,6 +77,19 @@ roslaunch webots_ros e_puck_line.launch
 ```
 This launch file will launch Webots (the `WEBOTS_HOME` environment variable should be set) and start the corresponding node.
 
+<details>
+<summary>If Webots is installed as a snap package the LD\_LIBRARY\_PATH has to be modified</summary>
+
+If Webots is installed as a snap package you need to append the `${WEBOTS_HOME}/projects/default/controllers/ros/lib/ros` path to the `LD_LIBRARY_PATH` environment variable (`export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${WEBOTS_HOME}/projects/default/controllers/ros/lib/ros`).
+
+```sh
+export WEBOTS_HOME=/snap/webots/x1/usr/share/webots/
```

Comment:
Many different version could exist it seems better to use `current` instead of `x1`
```suggestion
export WEBOTS_HOME=/snap/webots/current/usr/share/webots/
```